examples/regexp-type/regexp_type.bal

```diff
@@ -3,10 +3,16 @@ import ballerina/lang.regexp;
 
 public function main() returns error? {
     
-    // Declare a RegExp using regular expression literal.
-    string:RegExp _ = re `[a-zA-Z0-9]`;
+    // Declare a `RegExp` value using a regular expression literal.
+    regexp:RegExp alphanumericPattern = re `[a-zA-Z0-9]`;
+    // Matches any string that is a single alphanumeric character.
+    io:println("Pattern `[a-zA-Z0-9]` matches `a`: ", alphanumericPattern.isFullMatch("a"));
 
-    // Declare a RegExp using the `fromString` constructor method.
+    // `string:RegExp` is an alias for `regexp:RegExp`.
+    string:RegExp alphanumericPattern2 = re `[a-zA-Z0-9]`;
```

Comment:
Shall we define this in the first place, because this what we recommended

Reply:
I think this follows the description (The `RegExp` type is defined in the `lang.regexp` module, and can also be referenced using the type alias named the same in the `lang.string` module.) and natural flow given that it is an alias. 

The subsequent examples use `string:RegExp`, so this is okay IMO.